CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added a log collection guide ([#579](https://github.com/opensearch-project/opensearch-py/pull/579))
 - Added GHA release ([#614](https://github.com/opensearch-project/opensearch-py/pull/614))
 - Incorporated API generation into CI workflow and fixed 'generate' nox session ([#660](https://github.com/opensearch-project/opensearch-py/pull/660))
+- Added support for the `multi_terms` bucket aggregation
```

Comment:
Please add PR number here like
`Added support for the multi_terms bucket aggregation ([#663](https://github.com/opensearch-project/opensearch-py/pull/663))`